Sources/Support/Logging/LogRetriever.swift

```diff
@@ -1,11 +1,11 @@
 import Foundation
 
-public class LogRetriever {
+class LogRetriever {
```

Comment:
Current plan was to provide a public `LogRetriever` API so consumers can (if they choose to) use it without UI from the framework.

We need to see what would be the final stable UI though.